pyEpiabm/pyEpiabm/tests/test_unit/test_core/test_microcell.py

```diff
@@ -82,6 +82,19 @@ def test_set_location(self):
         self.assertRaises(ValueError, microcell.set_location, (1, (8, 6)))
         self.assertRaises(ValueError, microcell.set_location, ('1', 1))
 
+    def test_count_icu(self):
+        self.microcell.add_people(6)
+        for person in self.microcell.persons:
+            person.update_status(InfectionStatus(7))
+        self.assertEqual(self.microcell.count_icu(), 6)
```

Comment:
```suggestion
        self.microcell.add_people(6)
        for person in self.microcell.persons[:4]:
            person.update_status(InfectionStatus(7))
        self.assertEqual(self.microcell.count_icu(), 4)
```

What errors might this pick up that the original test doesn't?